packages/eventemittable/index.ts

```diff
@@ -1,22 +1,34 @@
 import compose from '@stamp/compose';
 import { EventEmitter } from 'events';
 
-interface ListenerCount {
-  (emitter: EventEmitter, event: string): number;
+import type { PropertyMap, Stamp } from '@stamp/compose';
+
+/**
+ * A Stamp with the EventEmitter behavior
+ */
+// TODO: EventEmitterStamp should support generics like <ObjectInstance, OriginalStamp>
+export interface EventEmitterStamp extends Omit<EventEmitter, 'listenerCount'>, Stamp {
+  /**
+   * Returns the number of listeners listening to the event named eventName
+   */
+  listenerCount: typeof listenerCount;
 }
 
-const listenerCount: ListenerCount = (emitter, event) => emitter.listenerCount(event);
+/** @internal */
+// ??? why emitter and not `this`
+const listenerCount = (emitter: EventEmitter, event: string): number => emitter.listenerCount(event);
```

Comment:
Why does `listenerCount` have  first parameter `emitter`? Shouldn't it be the `this`of the object instance? Neither README nor test suite provide sample/use case

Reply:
IDK :(